powerline/segments/common/players.py

```diff
@@ -101,6 +102,10 @@ def omitted_args(self, name, method):
 	+------------+-------------------------------------------------------------+
 	|total       |Composition length in format M:SS.                           |
 	+------------+-------------------------------------------------------------+
+	|file        |Mpd only option. Show file name of the song currently being  |
```

Comment:
I do not see why this must be MPD only, and this is not an option. Should be something like “(!No ‘show’, this is a parameter which exists, it cannot do any actions) File name of the song currently being played.” Plus some note in mpd documentation that “file” parameter is set to the file name with path relative to the default MPD music directory.